src/main/java/edu/kit/hci/soli/controller/LoginControllerAdvice.java

```diff
@@ -1,20 +1,32 @@
 package edu.kit.hci.soli.controller;
 
-import edu.kit.hci.soli.domain.LoginStateModel;
+import edu.kit.hci.soli.dto.LoginStateModel;
+import edu.kit.hci.soli.domain.User;
 import edu.kit.hci.soli.repository.VisitsRepository;
+import edu.kit.hci.soli.service.UserService;
 import org.springframework.security.core.annotation.AuthenticationPrincipal;
 import org.springframework.security.oauth2.core.oidc.user.OidcUser;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ModelAttribute;
 
 import java.security.Principal;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 
 @ControllerAdvice
 public class LoginControllerAdvice {
+
+    private final Logger logger = LoggerFactory.getLogger(LoginControllerAdvice.class.getName());
+
+
```

Comment:
Lombok provides `@Slf4j`, which is probably cleaner (at least if we are going to use lombok anyways)